lib/TwoFactorAuth.php

```diff
@@ -228,21 +218,21 @@ private function codeEquals(string $safe, string $user): bool
 
     private function getTime(?int $time = null): int
     {
-        return ($time === null) ? $this->getTimeProvider()->getTime() : $time;
+        return $time ?? $this->getTimeProvider()->getTime();
     }
 
     private function getTimeSlice(?int $time = null, int $offset = 0): int
     {
-        return (int) floor($time / $this->period) + ($offset * $this->period);
+        return (int)floor($time / $this->period) + ($offset * $this->period);
     }
 
     private function base32Decode(string $value): string
     {
-        if (strlen($value) == 0) {
+        if ($value === '') {
             return '';
         }
 
-        if (preg_match('/[^' . preg_quote(self::$_base32dict) . ']/', $value) !== 0) {
+        if (preg_match('/[^' . preg_quote(self::$_base32dict, '/') . ']/', $value) !== 0) {
```

Comment:
I don't think this change accomplishes anything unless you know PHP is planning to change the default delimiter?

Reply:
That's right, since `$_base32dict` doesn't contain any `/` character, it's a bit meaningless. I also don't think PHP is going to change the default delimiter anytime soon. Having it, just in case though, can't hurt, can it?